integration-tests/integration/reportPages/change-prison.spec.js

```diff
@@ -0,0 +1,118 @@
+    cy.task('stubPrison', 'LEI')
+    cy.task('stubPrisons')
+    cy.task('stubOffenders')
+    cy.task('stubUserDetailsRetrieval', 'MR_ZAGATO')
+    cy.task('stubUserDetailsRetrieval', 'MRS_JONES')
+    cy.task('stubUserDetailsRetrieval', 'TEST_USER')
+    cy.task('seedReport', { status: ReportStatus.IN_PROGRESS })
+  })
+
+  const fillFormAndSave = () => {
+    const reportUseOfForcePage = ReportUseOfForcePage.visit(bookingId)
+    const incidentDetailsPage = reportUseOfForcePage.startNewForm()
+    incidentDetailsPage.offenderName().contains('Norman Smith')
+    incidentDetailsPage.location().select('Asso A Wing')
+    incidentDetailsPage.forceType.check('true')
+    incidentDetailsPage.save()
+  }
+
+  const completeIncidentDetails = () => {
+    cy.login(bookingId)
+    fillFormAndSave()
+    cy.go('back')
+  }
+
+  it('Can edit prison prior to reaching check-your-answers page', () => {
+    completeIncidentDetails()
+
+    const incidentDetailsPage = IncidentDetailsPage.verifyOnPage()
+
+    incidentDetailsPage.clickChangePrison()
+
+    const changePrisonPage = ChangePrisonPage.verifyOnPage()
+
+    changePrisonPage.changePrisonToLeeds()
+
+    changePrisonPage.clickSave()
+
+    incidentDetailsPage.prison().contains('Leeds')
+
+    incidentDetailsPage.location().select('Asso A Wing')
+
+    incidentDetailsPage.saveAndReturn()
+
+    const reportUseOfForcePage = ReportUseOfForcePage.verifyOnPage()
+
+    reportUseOfForcePage.checkYourAnswersLink().click()
+
+    const checkAnswersPage = CheckAnswersPage.verifyOnPage()
+
+    checkAnswersPage.prison().contains('Leeds')
+  })
+
+  it('Can navigate from check-your-answers to incident-details and change prison', () => {
+    cy.login(bookingId)
+
+    const reportUseOfForcePage = ReportUseOfForcePage.visit(bookingId)
+
+    const checkAnswersPage = reportUseOfForcePage.goToAnswerPage()
+
+    checkAnswersPage.editIncidentDetailsLink().click()
+
+    const incidentDetailsPage = IncidentDetailsPage.verifyOnPage()
+
+    incidentDetailsPage.prison().contains('Moorland')
+
+    incidentDetailsPage.clickChangePrison()
+
+    const changePrisonPage = ChangePrisonPage.verifyOnPage()
+
+    changePrisonPage.changePrisonToLeeds()
+
+    changePrisonPage.clickSave()
+
+    incidentDetailsPage.prison().contains('Leeds')
+
+    incidentDetailsPage.location().select('Asso A Wing')
+
+    incidentDetailsPage.cancelButton().should('not.exist')
+
+    incidentDetailsPage.clickSave()
+
+    checkAnswersPage.prison().contains('Leeds')
+  })
+
+  it('Cancelling will not edit prison', () => {
+    completeIncidentDetails()
+
+    const incidentDetailsPage = IncidentDetailsPage.verifyOnPage()
+
+    incidentDetailsPage.prison().contains('Moorland')
+
+    incidentDetailsPage.clickChangePrison()
+
+    const changePrisonPage = ChangePrisonPage.verifyOnPage()
+
+    changePrisonPage.changePrisonToLeeds()
+
+    changePrisonPage.clickCancel()
+
+    incidentDetailsPage.prison().contains('Moorland')
```

Comment:
and here - if you move the const to a let you can reuse `incidentDetailsPage `

Reply:
done